virttest/env_process.py

```diff
@@ -563,12 +580,42 @@ def _call_image_func():
         else:
             process_images(image_func, test, params)
 
+    def _call_check_image_func():
```

Comment:
Try to reduce the redundant code between _call_image_func() and _call_check_image_func()

Reply:
Thanks.
This patch serves as a work around of the problem.
It needs to be refactored later on.